### PR TITLE
Task 38 contact layout

### PR DESCRIPTION
**Pull request by manhellis**
oh i should have done this in a new branch right oops im not sure

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -35,10 +35,18 @@
                 <div class="bun bottom"></div>
             </div>
         </header>
-        
-        <div class="contact-container flex-row">
-            <div class="imgContainer">
-                <img src="./img/Asset_9@2x.webp" alt="">
+        <main>
+        <h1>Contact Us</h1>
+        <div class=" flex-row">
```
Comment (alstdu) [inline contact.html]: unnecessary space before flex-row class

File: contact.html [hunk 2]
```diff
@@ -35,10 +35,18 @@
...
+            <div class="imgContainer flex-column">
+                <h2>We'd Love to Hear From You</h2>
+                <p>If you have any questions, concerns or want to get in touch, feel free to submit a message here!</p>
+                <!--<img src="./img/Asset_9@2x.webp" alt=""> -->
```
Comment (alstdu) [inline contact.html]: Commented out code - do we want this here ?

File: contact.html [hunk 3]
```diff
@@ -35,10 +35,18 @@
...
             </div>
         </header>
-        
-        <div class="contact-container flex-row">
-            <div class="imgContainer">
-                <img src="./img/Asset_9@2x.webp" alt="">
+        <main>
+        <h1>Contact Us</h1>
+        <div class=" flex-row">
+            <div class="imgContainer flex-column">
```
Comment (alstdu) [inline contact.html]: kabab-case for class names mayybbeee ?

File: index.html [hunk 4]
```diff
@@ -12,7 +12,7 @@
         <link rel="stylesheet" href="./styles/styles.css">
         <title>Home</title>
     </head>
-    <body class="flex-column">
+    <body class="contact-container">
```
Comment (alstdu) [inline index.html]: If index.html is going to have this class on body, it shouldn't be called contact-container

Was the change to index.html not intentional in this branch?

File: styles/contact.css [hunk 5]
```diff
@@ -11,24 +11,34 @@
     --white: #fff;
 }
 
+.contact-container{
```
Comment (alstdu) [inline styles/contact.css:14]: space after .contact-container and before { was removed

File: styles/contact.css [hunk 6]
```diff
@@ -11,24 +11,34 @@
...
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
 
-.contact-container {
+}
+/* .contact-container {
```
Comment (alstdu) [inline styles/contact.css]: commented out code - do we still need this? should we label what it is if we are going to keep it commented out and here?

**Review by alstdu — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (manhellis) [reply index.html]: it was a mistake, i thought i had removed it but i guess notor maybe i still dont know how to use git well.

**Comment by manhellis** [comment]
ok i fixed the issues and pushed it

File: contact.html [hunk 7]
```diff
@@ -55,13 +63,20 @@
                 </div>
                 <div class="inputRow">
                     <!-- Send button -->
-                    <button type="submit">Send</button>
+                    <button class="button filled-button fakeAlert" type=submit>Send</button>
```
Comment (alstdu) [inline contact.html]: We maybe shouldn't remove the quotes around "submit". Idk if it'll actually break anything according to Google, but they were there before so I'm just gonna add them back.

**Review by alstdu — APPROVED** [review]
Thanks so much :)

**Pull request merged**